__sigemptyset has been removed from glibc public
API headers in upcoming (2.26) release onwards

Upstream-PR: https://github.com/telmich/gpm/pull/20

Index: gpm-1.20.7/src/daemon/open_console.c
===================================================================
--- gpm-1.20.7.orig/src/daemon/open_console.c
+++ gpm-1.20.7/src/daemon/open_console.c
@@ -22,6 +22,7 @@
 #include <fcntl.h>                  /* open and co.      */
 #include <sys/stat.h>               /* stat()            */
 #include <sys/ioctl.h>              /* ioctl             */
+#include <sys/sysmacros.h>	    /* major */
 
 /* Linux specific (to be outsourced in gpm2 */
 #include <linux/serial.h>           /* for serial console check */
Index: gpm-1.20.7/src/prog/gpm-root.y
===================================================================
--- gpm-1.20.7.orig/src/prog/gpm-root.y
+++ gpm-1.20.7/src/prog/gpm-root.y
@@ -1196,7 +1196,7 @@ int main(int argc, char **argv)
    /* reap your zombies */
    childaction.sa_handler=reap_children;
 #if defined(__GLIBC__)
-   __sigemptyset(&childaction.sa_mask);
+   sigemptyset(&childaction.sa_mask);
 #else /* __GLIBC__ */
    childaction.sa_mask=0;
 #endif /* __GLIBC__ */
